### ISSUE TITLE
Suggestion: auto update built-in app

### ISSUE THREAD

**Issue by DoggoNako** [JustYuuto/deezer-discord-rpc#25]
Hi !
The title says everything, I know there's already something to check if there's an update but is it possible to install automatically ? 
This can surely be useful: https://www.electronjs.org/docs/latest/api/auto-updater
I've just read some and you might need to sign your app and there's nothing for Linux...

**Comment by JustYuuto** [comment]
Hi!
Windows and macOS code signing requires annual fees which I'm obviously not going to pay, so Squirrel will not work with those platforms, and Linux doesn't work with Squirrel as written in the docs...
So I'm sorry but if you have alternatives solutions feel free to propose them

**Comment by DoggoNako** [comment]
Hey! 
Yeah it hurt the price for just a code signature... 
I've found that [Squirrel](https://github.com/Squirrel/Squirrel.Windows) is not updated since 3 years... (at least for the windows version)
But I've found something that's it seems to be used a lot around here:
https://www.electron.build

About disabling code signature:
https://github.com/electron-userland/electron-builder/issues/144

About auto update without code signature:
https://stackoverflow.com/questions/42286648/is-it-possible-to-auto-update-electron-apps-without-a-certificate

For Windows it seems working for everyone without code sign but for Mac it's either working with extra steps from user side or just not... and heeey ! Linux is supported. (Better two than one lol).
